src/main/scala/org/scalajs/cli/Scalajsld.scala

```diff
@@ -81,15 +100,24 @@ object Scalajsld {
         .text("Execute the specified main(Array[String]) method on startup")
       opt[File]('o', "output")
         .valueName("<file>")
-        .required()
-        .action { (x, c) => c.copy(output = x) }
-        .text("Output file of linker (required)")
+        .action { (x, c) => c.copy(output = Some(x)) }
+        .text("Output file of linker (deprecated)")
+      opt[File]("outputDir")
+        .valueName("<file>")
+        .action { (x, c) => c.copy(outputDir = Some(x)) }
+        .text("Output directory of linker (required)")
       opt[Unit]('f', "fastOpt")
         .action { (_, c) => c.copy(noOpt = false, fullOpt = false) }
         .text("Optimize code (this is the default)")
       opt[Unit]('n', "noOpt")
         .action { (_, c) => c.copy(noOpt = true, fullOpt = false) }
         .text("Don't optimize code")
+      opt[ModuleSplitStyle]("moduleSplitStyle")
+        .action { (x, c) => c.copy(moduleSplitStyle = x) }
+        .text("Module splitting strategy " + ModuleSplitStyleRead.All.mkString("(", ", ", ")"))
+      opt[String]("outputPatterns")
```

Comment:
call this `jsFilePattern`? I think for now it is OK to only support this specific type of pattern, but we should reserve the `outputPatterns` name for that option, if we ever want to introduce it.